selftests/functional/plugin/spawners/test_process.py

```diff
@@ -20,12 +20,11 @@ def test_logdir_path(self):
         test = script.Script(os.path.join(self.tmpdir.name, "logdir_test.py"),
                              TEST_LOGDIR)
         test.save()
-        result = process.run("%s run --job-results-dir %s --disable-sysinfo "
-                             "--json - -- %s" % (AVOCADO, self.tmpdir.name,
-                                                 test))
+        result = process.run(f"{AVOCADO} run --job-results-dir {self.tmpdir.name}"
```

Comment:
When I made my comment, I was talking about this one: 77 vs 82.

Reply:
sigh, I was mistaken by the fact that we run pylint *twice*  with different options in `selftests/lint.sh `